packages/beacon-node/src/chain/chain.ts

```diff
@@ -29,6 +29,7 @@ import {
   isBlindedBeaconBlock,
   BeaconBlock,
   SignedBeaconBlock,
+  FullOrBlindedSignedBeaconBlock,
```

Comment:
Why did we reintroduce this type? Based on discussion in type refactor we don't want this type